scripts/ansible/roles/linux/openenclave/vars/ubuntu/focal.yml

```diff
@@ -3,4 +3,4 @@
 
 ---
 gcc_target_version: "9.3.0"
-clang_target_version: "8.0.1"
+clang_target_version: "10.0.0"
```

Comment:
@CyanDevs are we good bumping this up to 10.0.1?

Reply:
It looks like the default clang-10 version is 10.0.1 so it makes sense to.
```
$ clang-10 --version
Ubuntu clang version 10.0.1-++20210313014558+ef32c611aa21-1~exp1~20210313125203.63
Target: x86_64-pc-linux-gnu
Thread model: posix
InstalledDir: /usr/bin
```